src/hooks/use-text-area-input.hook.ts

```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react';
```

Comment:
this is nice! worth going back and putting it in the nhs app create page as well?

Reply:
Ah, I thought i had! Clearly I'd forgotten.

Reply:
Updated.